PetAdoption-iOS/PetAdoption-iOS.xcodeproj/project.pbxproj

```diff
@@ -1188,7 +1188,7 @@
 				);
 				INFOPLIST_FILE = "PetAdoption-iOS/Info.plist";
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
-				PRODUCT_BUNDLE_IDENTIFIER = com.cfo.petadoption;
+				PRODUCT_BUNDLE_IDENTIFIER = com.lhaquebaruma.petadoption;
```

Comment:
Looks like you forgot to set the bundle identifier back again heh

Reply:
I'll be sure to set it to cfo before I push the updates to git